grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -978,6 +978,11 @@ class OrderController {
 
     def createCombinedShipment = {
         def orderInstance = Order.get(params.orderId)
+        if (!orderInstance.orderItems.find {it.quantityRemainingToShip != 0 }) {
+            flash.error = "${warehouse.message(code:'purchaseOrder.noItemsToShip.label')}"
```

Comment:
Just use flash.message. It's more warning / informational and not really an error anyway.